src/bcs/serializable/move-structs.ts

```diff
@@ -439,7 +439,7 @@ export class MoveObject extends Serializable implements TransactionArgument {
     if (value instanceof AccountAddress) {
       this.value = value;
     } else {
-      this.value = AccountAddress.fromHexInput({ input: value });
+      this.value = AccountAddress.fromHexInputRelaxed({ input: value });
```

Comment:
Why is this changing?

Reply:
I would like to use relaxed if we can, I was running into a bunch of errors because of it- basically what we were talking about with daniel

I was just putting it here as a fix so I didn't have to fix every instance of an object in the test suite (I wrote it down as feedback/things to think about in the doc), but yea, the issue was I was getting errors from addresses I was getting from the API on chain